internal/server/config.go

```diff
@@ -42,6 +42,10 @@ type Config struct {
 	// to access RulesPath files
 	RulesPathSecret string `yaml:"ruleset_secret" mapstructure:"ruleset_secret"`
 
+	// Type of storage to where ruleset definition is stored
+	// The value must be set to either BLOB or DB
+	RulesConfigStorage string `yaml:"rules_config_storage" mapstructure:"rules_config_storage" default:"blob"`
```

Comment:
Lets use the existin `RulesPath` regardless it is file or DB. We can define `postgresql://localhost` as an identifier if it is using local DB